src/types/chat/message/interfaces/message.interface.ts

```diff
@@ -15,7 +17,13 @@ export interface MessageInterface extends CommonEntityFields {
 
 export interface GetMessagesParams
   extends Partial<Omit<RequestParams, 'sort'>> {
-  chatId: string
+  chatId: Pick<ChatResponse, '_id'>
+}
+
+export interface TextAreaSx {
```

Comment:
I don`t think you need to create an interface for that, it will be used only in ChatTextArea component

Reply:
Fixed